languages/julia/concept-exercises/multiple-dispatch/.docs/instructions.md

```diff
@@ -31,7 +31,7 @@ First, add an abstract type `Pet` to your implementation, of which `Cat` and `Do
 
 - If a pet meets another pet that it doesn't recognize, it **is cautious**.
 
-There are many other things that pets may encounter that aren't pets: cars, humans, plants, natural disasters, asteroids…. What happens then?
+There are many other things that pets may encounter that aren't pets: cars, humans, plants, natural disasters, asteroids... What happens then?
```

Comment:
```suggestion
There are many other things that pets may encounter that aren't pets: cars, humans, plants, natural disasters, asteroids…. What happens then?
```

Unless there's a strict requirement for files to be ASCII-only, I'd prefer if this wasn't changed. There will be many more files that require unicode in the Julia track anyway, so I don't see the point in this change.

Reply:
The use of both the ellipsis unicode character *and* a trailing period (`….`)is incorrect. It should be changed to either:
- Three periods to compose an ASCII ellipsis (`...`) (current change)
- The ellipsis character sans-trailing period (`…`)

Either is fine with me.

Reply:
Ah, I didn't know that. Thanks for the correction. Let's go with the 2nd option then.

Reply:
I've updated the `suggestion`-block above. Apparently it's not possible to have two suggestions in one comment thread